jboss-kie-common/configure.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+# Openshift JBoss KIE - Common scripts and helpers
+set -e
+
+SCRIPT_DIR=$(dirname $0)
+ADDED_DIR=${SCRIPT_DIR}/added
+
+# Add common scripts/libraries/snippets
+mkdir -p ${JBOSS_HOME}/bin/launch
```

Comment:
For the JBOSS_HOME env var to be usable here, you will need to add it to the rhpam-7-image/smartrouter/modules/smartrouter/module.yaml file in the envs section, as that env var won't be available to you by default in non-EAP based image builds otherwise.

Reply:
Also please squash all these commits into one.